### ISSUE TITLE
Synonyms

### ISSUE THREAD

**Issue by scratchpads** [NaturalHistoryMuseum/scratchpads2#5940]
- Added by [Christos Georgiadis](http://antsme.myspecies.info/user/2)
- Scratchpad: [Ants of the Middle East](http://antsme.myspecies.info)

### Description:

Is there a way to show synonymic data under the valid taxon? Or at least, at the Overview tab to have a direct link to the valid taxon?

**Comment by PeterHovenkamp** [comment]
Is this the same issue as #5848?

**Comment by Chrisis21** [comment]
Not really... I am saying that when you designate a synonym (or any invalid reason...) it would be nice to automatically put the invalid taxa under the valid taxon (maybe with a = sign or something...). E.g.

Taxon A is junior synonym of Taxon B so at the taxonomy tree it would show as
+ Taxon B
  = Taxon A

or at least in the Description tab of Taxon A it should have a link to Taxon B.
